### PR TITLE
bump sts version

### PR DESCRIPTION
**Pull request by cssuh**
https://github.com/microsoft/sqltoolsservice/releases/tag/v3.0.0-release.93

Contains this PR: https://github.com/microsoft/sqltoolsservice/pull/1190 which I need to filter some of the DB migration warnings

### CODE REVIEW

File: extensions/kusto/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.92",
+	"version": "3.0.0-release.93",
```
Comment (kburtram) [inline extensions/kusto/config.json:3]: why are we bumping the version in the Kusto extension for SQL DB migrations?

**Review by kburtram — CHANGES_REQUESTED** [review]
Not sure if I'm missing something, but I bumped the MSSQL STS earlier today to pickup a db pool fix with https://github.com/microsoft/azuredatastudio/pull/15163/files.

File: extensions/kusto/config.json (hunk 1 above)
Comment (cssuh) [reply extensions/kusto/config.json:3]: oh I missed that, sorry! I must have accidentally changed the wrong config file

**Comment by cssuh** [comment]
not needed because karl has bumped the version

**Pull request closed without merging**